app/routes/wallet/credentials.py

```diff
@@ -136,38 +235,96 @@ async def get_credential_revocation_status(
     return result
 
 
-@router.get("/w3c", response_model=VCRecordList)
+@router.get(
+    "/list/w3c",
+    response_model=VCRecordList,
+    summary="Fetch a list of W3C credentials from the wallet",
+)
 async def list_w3c_credentials(
-    count: Optional[str] = None,
-    start: Optional[str] = None,
-    wql: Optional[str] = None,
-    body: Optional[W3CCredentialsListRequest] = None,
+    contexts: Optional[List[str]] = Query(None),
+    types: Optional[List[str]] = Query(None),
+    schema_ids: Optional[List[str]] = Query(None),
+    subject_ids: Optional[List[str]] = Query(None),
+    proof_types: Optional[List[str]] = Query(None),
+    issuer_id: Optional[str] = Query(None),
+    given_id: Optional[str] = Query(None),
+    tag_query: Optional[str] = Query(None),
+    max_results: Optional[int] = Query(None),
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> VCRecordList:
-    """Fetch a list of W3C credentials from the wallet."""
+    """
+    Fetch a list of W3C credentials from the wallet
+    ---
+
+    The W3C credentials can be filtered by the parameters provided.
+
+    Optional Parameters:
+    ---
+        contexts: List[str]
+        types: List[str]
+        schema_ids: List[str]
+        issuer_id: str
+        subject_ids: List[str]
+        given_id: str
+        proof_types: List[str]
+        tag_query: str
+        max_results: int
```

Comment:
After the options are trimmed, we can just add a quick description for each one we keep here.